src/main/java/com/bynder/sdk/util/Utils.java

```diff
@@ -99,17 +101,37 @@ private static OkHttpOAuthConsumer createHttpOAuthConsumer(final String consumer
      * @return {@link OkHttpClient} instance used for API requests.
      */
     private static OkHttpClient createHttpClient(final OkHttpOAuthConsumer consumer) {
+        return createHttpClient(consumer, null);
+    }
+
+    /**
+     * Creates an instance of {@link OkHttpClient}.
+     *
+     * @param consumer {@link OkHttpOAuthConsumer} instance.
+     * @param httpConnectionSettings Settings for the http connection to Bynder
+     *
+     * @return {@link OkHttpClient} instance used for API requests.
+     */
+    private static OkHttpClient createHttpClient(final OkHttpOAuthConsumer consumer, HttpConnectionSettings httpConnectionSettings) {
         OkHttpClient.Builder httpClient = new OkHttpClient.Builder();
         httpClient.interceptors().clear();
         httpClient.addInterceptor(new SigningInterceptor(consumer));
 
         HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
         interceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
         httpClient.addInterceptor(interceptor);
-
-        httpClient.readTimeout(30, TimeUnit.SECONDS);
-        httpClient.connectTimeout(30, TimeUnit.SECONDS);
-
+        if (httpConnectionSettings == null) {
```

Comment:
you don't need this if, by default the httpConnectionSettings will already be instantiated with the empty constructor